src/EnergyPlus/RoomAirModelManager.cc

```diff
@@ -2676,7 +2697,7 @@ namespace RoomAir {
                         std::string const &EquipName, // Equipment Name
                         std::string &SupplyNodeName,  // Supply node name
                         std::string &ReturnNodeName,  // Return node name
```

Comment:
In one of your later phases, these should be changed to NodeNum instead of passing and saving node names.